entrypoint.py

```diff
@@ -37,10 +37,16 @@ def setup_env(
         Path("/tmp/gcs_keyfile.json").write_text(gcs_keyfile)
 
 
-def install_edr(adapter: str, project_dir: Optional[str]):
+def install_edr(
+    adapter: str, project_dir: Optional[str], profile_target: Optional[str]
+):
     logging.info("Getting Elementary dbt package version.")
     try:
         dbt_pkg_ver = None
+        possible_target_flag = (
```

Comment:
What do you think about exporting the arguments passed to `subproccess.run()` to a `command_args` list and then do:

```py
if profile_target:
    command_args.extend(["--target", profile_target])
```

Here's [an example](https://github.com/elementary-data/elementary/blob/45db8d8d8b1ea3f761eadcaccd25b3044cf2de29/elementary/clients/dbt/dbt_runner.py#L146).

Reply:
I've applied that pattern, thanks 👍🏻